guides/client/js-interop.md

```diff
@@ -28,6 +28,15 @@ except for the following LiveView specific options:
     See the [Key events](bindings.html#key-events) section in the bindings guide
     for an example.
 
+The `liveSocket` instance exposes the following methods:
```

Comment:
Not strictly related, but seems like the `liveSocket` API has now grown to a size where a list of all methods is helpful.